src/app/modules/shared/utils/project-storage.util.ts

```diff
@@ -0,0 +1,30 @@
+import { Project } from '../models/project.model';
+import { PROJECTS_KEY_FOR_LOCAL_STORAGE } from '../../../../environments/environment';
+
+const projectsKey = PROJECTS_KEY_FOR_LOCAL_STORAGE;
```

Comment:
I think is not necessary define a constant to this, because you can use directly the value in your code, if you want change the name you could use import as to change the name or define it as readonly inside your function

Reply:
That's a good suggestion Gustavo, I forgot the as inside the import, I will implement in that form, just a clarification, as you can see we are using functions, and the readonly is only available at the class level